tools/sdk-generation-pipeline/ci.yml

```diff
@@ -130,16 +130,16 @@ stages:
             downloadPath: '$(VAR_BUILD_ARTIFACT_STAGING_DIRECTORY)'
 
         - script: |
-            echo -e "\e[32m[$(date -u)] LOG: Ensure latest npm is installed"
-            sudo npm install -g npm || { echo 'npm install failed' ; exit 1; }
+            echo -e "\e[32m[$(date -u)] LOG: Ensure npm is installed"
+            sudo npm install -g npm@6.14.16 || { echo 'npm install failed' ; exit 1; }
```

Comment:
I generally like pinning tools versions but just for my context what ended up breaking because of the latest npm version? Also Is there a reason that the npm version that is already on the agent machine doesn't work?